### DISCUSSION TITLE
Seismic properties map

### DISCUSSION THREAD

**Discussion by prastyani28** [mtex-toolbox/mtex#1403] [Ask Anything] [answered]
Hi all,

I'm still new to MTEX and need more guidance :)

I have plotted a seismic properties map from my ebsd data set.

![Hill_bulk](https://user-images.githubusercontent.com/67193615/163830798-633ad394-343b-44fb-ac1b-959ed92d850f.jpg)

But, when I looked at the map, I think the information given below each map and the color bar is a bit mismatched.
For example, for the Vp, the color bar shows that the maximum Vp is around 6.9 while the information below the Vp map is 15.0.
Is this what it is? Or does anyone know that I need to adjust anything from my script?

I used this code to plot the seismic properties:

```
plotSeismicVelocities(CHill)

```

Could anyone also give me references on how to calculate the anisotropy in this map in MTEX?

Thanks so much for the attention and time!

Best,
Erina

**Comment by lfgmorales** [comment, marked as answer]
Hi

If you look in detail, it states in the figure that the value of 15.0 refers to the anisotropy of P-waves, which is given by 200 (Vmax-Vmin)/(Vmax+Vmin), if you want to know the max and min values, you need the option ‘minmax’

Maybe you should start with

http://www.gm.univ-montp2.fr/PERSO/mainprice/W_data/My_Publications_PDF/Mainprice15ChapterAniso2ndEdition.pdf

https://hal.archives-ouvertes.fr/hal-01685568/file/Almqvist_et_al-2017-Reviews_of_Geophysics.pdf

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 18 Apr 2022, at 17:27, NanaAcademics ***@***.***> wrote:

﻿

Hi all,

I have plotted a seismic properties map from my ebsd data set.

[Hill_bulk]<https://user-images.githubusercontent.com/67193615/163830798-633ad394-343b-44fb-ac1b-959ed92d850f.jpg>

But, when I looked at the map, I think the information given below each map and the color bar is a bit mismatched.
For example, for the Vp, the color bar shows that the maximum Vp is around 6.9 while the information below the Vp map is 15.0.
Is this what it is? Or does anyone know that I need to adjust anything from my script?

I used this code to plot the seismic properties:

plotSeismicVelocities(CHill)



Could anyone also give me references on how to read the seismic properties map in MTEX?

Thanks so much for the attention and time!

Best,
Erina

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1403>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INYLHM5E6WXC5QNJSHTVFV5PPANCNFSM5TWEHJFQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1403", "url": "https://github.com/mtex-toolbox/mtex/discussions/1403", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by prastyani28** [reply]
Hi!

thanks so much for your answer and also the references that you have attached :)

Best,
Erina

**Reply by prastyani28** [reply]
Hi again @lfgmorales ,

I have read the references that you have given to me, but I didn't find the formula to calculate the anisotropy as you have mentioned.
Where did you get the formula of 200 (Vmax-Vmin)/(Vmax+Vmin) to calculate the anisotropy?

Thanks again! :)

Best,
Erina

**Reply by marcoalopez** [reply]
check page 506 in http://www.gm.univ-montp2.fr/PERSO/mainprice/W_data/My_Publications_PDF/Mainprice15ChapterAniso2ndEdition.pdf or page 422 in https://hal.archives-ouvertes.fr/hal-01685568/file/Almqvist_et_al-2017-Reviews_of_Geophysics.pdf when authors define seismic anisotropy.

Cheers.
Marco

**Reply by prastyani28** [reply]
Hi, thanks so much!

My bad, I didn't check until the glossary part 😅

Best,
Erina
